Tools/setup/ubuntu.sh

```diff
@@ -115,15 +116,15 @@ fi
 # Python3 dependencies
 echo
 echo "Installing PX4 Python3 dependencies"
-sudo python3 -m pip install --upgrade pip setuptools wheel
-sudo python3 -m pip install -r ${DIR}/requirements.txt
+pip3 install --user --upgrade setuptools wheel
```

Comment:
PATH update if we're going to go with the `--user` install? I know this was always the "right" way to do it, but it always seemed to cause more confusion then just installing system wide.

Reply:
I've always done it this way and don't remember having problems. Which tools do we need from the user directory that require the path to be added?

Reply:
I'll try again on a fresh ubuntu setup. In the past I recall needing to set the PYTHONPATH to find things installed with `--user`. The other question is what happens in docker when this is run as root.